packages/datadog-plugin-cypress/src/index.js

```diff
@@ -0,0 +1,6 @@
+module.exports = [
```

Comment:
yeah I'll do that

Reply:
Ah yes I was thinking that externals.json could be used for this but it needs at least the version of the actual library to get started. I guess it's fine then, it will simply never be loaded.

Reply:
I moved it to the test file in https://github.com/DataDog/dd-trace-js/commit/a1d54686ca5ab26096aac2396220051a354e2a07, but if you think it works there I can revert it

Reply:
Fine either way so let's keep it in the test 👍